plugins/inputs/procstat/procstat.go

```diff
@@ -74,7 +81,17 @@ func (_ *Procstat) Description() string {
 
 func (p *Procstat) Gather(acc telegraf.Accumulator) error {
 	if p.createPIDFinder == nil {
-		p.createPIDFinder = defaultPIDFinder
+		fmt.Println("PIDFINDER: ", p.PidFinder)
+		switch p.PidFinder {
+		case "native":
+			fmt.Println("pid finder is now native")
```

Comment:
Don't forget to cleanup debugging statements

Reply:
done